Update SQL Server and Elastic Search README docs (#186)
hasura/ndc-hub: registry/sqlserver/README.md

CONNECTION_URI=<YOUR_SQLSERVER_URL>
```

### 3. Introspect your indices

> From the root of your project run:

Reviewer: ```suggestion
Run the following command from your project's root directory:
```

### 4. Create the Hasura metadata

> Run the following from the root of your project:

Reviewer: ```suggestion
Run the following command from your project's root directory:
```

### 5. Start the connector's docker compose

> Let's start our connector's docker compose file. Run the following from the connector's subdirectory inside a subgraph:

Reviewer: Unclear why we need this "inside a subgraph" - isn't the connector's subdirectory good enough?

```

Now, whenever running the following, you'll bring up the GraphQL engine, observability tools, and any connectors you've

> included. From the root of your project, run:

Reviewer: ```suggestion
included. Run the following command from your project's root directory:
```

Finally, now that our `DataConnectorLink` has the correct environment variables configured for the SQL Server connector,
we can run the update command to have the CLI look at the configuration JSON and transform it to reflect our database's

> schema in `hml` format. In a new terminal tab from the root of your project, run:

Reviewer: ```suggestion
schema in `hml` format. In a new terminal, from your project's root directory, run:
```

### 8. Import _all_ your indices

> You can do this in one convenience command. From the root of your project, run:

Reviewer: ```suggestion
You can do this with just one command. Run the following command from your project's root directory:
```
